src/Controller/ListPageRssController.php

```diff
@@ -164,12 +192,57 @@ public function build(NodeInterface $node): CacheableResponse {
    *
    * @param \Drupal\node\NodeInterface $node
    *   The node containing the list.
+   * @param \Drupal\Core\Cache\CacheableMetadata $cache_metadata
+   *   The current cache metadata.
    *
    * @return array
    *   Array of items to be rendered.
    */
-  protected function getItemList(NodeInterface $node): array {
-    return [];
+  protected function getItemList(NodeInterface $node, CacheableMetadata $cache_metadata): array {
+    $configuration = ListPageConfiguration::fromEntity($node);
+    $execution_result = $this->listExecutionManager->executeList($configuration);
+    $query = $execution_result->getQuery();
+    $results = $execution_result->getResults();
+    $cache_metadata->addCacheableDependency($query);
+    $cache_metadata->addCacheableDependency($results);
+    $cache_metadata->addCacheTags(['search_api_list:' . $query->getIndex()->id()]);
+    $result_items = [];
+    foreach ($results->getResultItems() as $item) {
+      /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+      $entity = $item->getOriginalObject()->getEntity();
+      $cache_metadata->addCacheableDependency($entity);
+      $creation_date = $entity->get('changed')->value;
+      if ($entity->hasField('body')) {
+        $description = $entity->body->view([
+          'type' => 'text_default',
+          'label' => 'hidden',
+          'settings' => [],
+        ]);
+        $description = Html::escape($this->renderer->renderPlain($description[0]));
```

Comment:
Why you escaping here again? Twig already escapes no?

Seems we get the same result in Views so let's add a comment here as to why we are double escaping.